src/main/scala/ru/otus/sc/counter/dao/impl/CounterDaoImpl.scala

```diff
@@ -0,0 +1,17 @@
+package ru.otus.sc.counter.dao.impl
+
+import ru.otus.sc.counter.dao.CounterDao
+
+class CounterDaoImpl extends CounterDao {
+  private var count: Long = 0
+
+  def clearCount: String = {
+    count = 0
+    getCount
+  }
```

Comment:
```suggestion
  def clearCount: Unit = {
    count = 0
  }
```

Я бы оставил `Unit` чтобы быть более специфичным о том, что этот метов делает. Он сбрасывает счётчик, `getCount` вовзращает значение.

Reply:
хорошо